services/version-api/helpers/getApplicationUpdateStatus.js

```diff
@@ -0,0 +1,34 @@
+import smCompare from 'semver-compare';
+
+import { VERSION_STATUS } from '../constants';
+
+const COMPARISON_RESULT = {
+  EQUAL: 0,
+  SMALLER: -1,
+  GREATER: 1,
+};
+
+/**
+ * @param {{current: string, min: string, max: string}} parameters (properties in semver format i.e "1.2.0")
+ * @returns {string} Status
+ */
+function getApplicationUpdateStatus({ current, min, max }) {
+  const maxComparison = smCompare(current, max);
+  const minComparison = smCompare(current, min);
```

Comment:
Could we rename these to something like `currentComparedToMax` and `currentComparedToMin` respectively? My intention is to make it more clear as when reading the following `if` statement i'm not sure what result to expect.

```javascript
if(currentComparedToMax === COMPARISION_RESULT.SMALLER)
```

Reads more clearly to me as "if current version is smaller than max".

Reply:
I remember I had difficulties naming this constants, but I like the idea about naming them to `currentComparedToMax ` and `currentComparedToMin `. Will do that change